CMakeLists.txt

```diff
@@ -41,6 +87,9 @@ else ()
   endif()
```

Comment:
nit above: we're not bundling zstd (dont need to fix it in this PR)

Reply:
If I don't fix it now, I will eventually forget about it :grinning:

Reply:
Fixed in https://github.com/edenhill/librdkafka/pull/2262/commits/1d0fadb1a9e9f42fc4b0674decb86d9b9704252e.